package/kernel/linux/modules/netdevices.mk

```diff
@@ -538,6 +538,56 @@ endef
 
 $(eval $(call KernelPackage,dsa))
 
+
+define KernelPackage/dsa-tag-brcm
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Broadcom DSA tagging support
+  DEPENDS:=+kmod-dsa
+  KCONFIG:= CONFIG_NET_DSA_TAG_BRCM_COMMON
+  FILES:=$(LINUX_DIR)/net/dsa/tag_brcm.ko
+  AUTOLOAD:=$(call AutoLoad,40,tag_brcm,1)
+endef
+
+define KernelPackage/dsa-tag-brcm/description
+  Kernel module for Broadcom DSA tagging
+endef
+
+$(eval $(call KernelPackage,dsa-tag-brcm))
+
+
+define KernelPackage/dsa-b53
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Broadcom BCM53xx managed switch DSA support
+  DEPENDS:=+kmod-dsa-tag-brcm
+  KCONFIG:=CONFIG_B53 \
+  CONFIG_NET_DSA_TAG_BRCM=y
```

Comment:
```suggestion
```
Please do not compile `CONFIG_NET_DSA_TAG_BRCM` into the kernel, you should select it as a module in `kmod-dsa-tag-brcm`.

Reply:
Completely clear, Hauke. As discussed [above](CONFIG_NET_DSA_TAG_BRCM_COMMON), I'm not doing this anymore.

Thanks for the review, I really appreciate the explanations.